### ISSUE TITLE
disassembled mnemonics for 0xed 0xa3 should be outi

### ISSUE THREAD

**Issue by mortenjc** [kosarev/z80#61]
One more of the ed 'escape' mnemonics :-)

```
11c5 ed           ; db 0xed              
11c6 a3           ; and e
```
should be

`
11c5 ed a3       ; outi            
`

**Comment by kosarev** [comment]
Resolved via 7ddaa4cb7512aa7b1abb1f7cd8f2ad8cd8e37558.

**Comment by kosarev** [comment]
Closing this as resolved, but please re-open if something.
